packages/framework/src/Model/Pricing/BasePriceCalculation.php

```diff
@@ -28,41 +28,26 @@ public function __construct(
      * @param \Shopsys\FrameworkBundle\Model\Pricing\Currency\Currency $currency
      * @return \Shopsys\FrameworkBundle\Model\Pricing\PriceInterface
      */
-    public function calculateBasePriceRoundedByCurrency(
+    public function calculateRoundedBasePrice(
```

Comment:
I have a feeling that the method name could remain the same, that would mean one BC break less, but I understand if you do not want to change it back :smile: But let's discuss it please

Reply:
For me this change is a must. I think the naming was misleading, as only price with VAT is rounded by currency, but price without VAT is not.